optaplanner-core/src/main/java/org/optaplanner/core/api/score/stream/ConstraintStreamImplType.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.core.api.score.stream;
+
+public enum ConstraintStreamImplType {
```

Comment:
We should add some Javadocs here, specifically:

- that both implementations are incomplete,
- and that for the foreseeable future, we would be spending our energy on the Drools impl.

Reply:
I'd leave that out of the javadocs, as it's too inaccurate for javadocs. We can mention it in the docs. (far more people read the docs than the javadocs anyway)